.github/workflows/build.yml

```diff
@@ -0,0 +1,96 @@
+name: Build
+
+on:
+  push:
+    branches: [ "main" ]
+    tags:
+      - '*'
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set version name  # ref_name but replace slashes and spaces with dashes
+      id: version
+      run: echo "ref-name=$(echo $GITHUB_REF_NAME | sed 's/\//-/g' | sed 's/ /-/g')" >> $GITHUB_OUTPUT
+    
+    - name: checkout code
+      uses: actions/checkout@v4
+      with:
+        submodules: true
+            
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 8.0.x
+
+    - name: Test
+      run: dotnet test --no-build --verbosity normal
+

```

Comment:
Lumper doesn't currently contain any tests, unlikely we add them in the near future unless dev activity really picks up after 0.10.0.